internal/git/gitService.go

```diff
@@ -91,12 +91,17 @@ func setUpLocalGitFolder(folderName string, clusterRootPath string, clusterName
 
 func moveCloneIntoLocalRepo(folderName string, clusterRootPath string) {
 
-	commandRmGit := exec.Command("rm", "-rf", ".git")
-	commandRmGit.Dir = clusterRootPath + FPS + "tmp" + FPS + folderName
-	errRmGit := commandRmGit.Run()
-	if errRmGit != nil {
-		log.Fatalf("Error removing git folder at /tmp/"+folderName+". %s", errRmGit)
-	}
+	// commandRmGit := exec.Command("rm", "-rf", ".git")
+	// commandRmGit.Dir = clusterRootPath + FPS + "tmp" + FPS + folderName
+	// errRmGit := commandRmGit.Run()
+	// if errRmGit != nil {
+	// 	log.Fatalf("Error removing git folder at /tmp/"+folderName+". %s", errRmGit)
+	// }
+
+	removeGitError := os.RemoveAll(clusterRootPath + FPS + "tmp" + FPS + folderName + FPS + ".git")
```

Comment:
There is a native way in go to deal with os specific file path creation. `filepath.Join()`

I have refactored all of the FPS variable usage out of the code in a PR I wrote a while ago but wasn't merged yet. still a little WIP. #18